README.md

```diff
@@ -51,6 +51,7 @@ zfs | Exposes [ZFS](http://open-zfs.org/) performance statistics. | [Linux](http
 
 Name     | Description | OS
 ---------|-------------|----
+bcache | Exposes bcache statistics | Linux
```

Comment:
This wouldn't work well right now without gracefully handling systems without bcache.